### DISCUSSION TITLE
Get blame of diff in pull request.

### DISCUSSION THREAD

**Discussion by heckad** [community/community#24867] [Apps, API and Webhooks]
<p>How to get the blame of diff and diff using graphql API? In documentation didn`t find information about gatting blame in a pull request.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>From the <code>PullRequest</code> object you would use the <code>commits</code> connection to get the list of <code>PullRequestCommit</code> objects. From each of those, you can use the <code>commit</code> field to get the <code>Commit</code> object. And from the <code>Commit</code> object, you can get the blame information through the <code>blame</code> field.</p>
<p>I hope that helps!</p>
